tests/base.py

```diff
@@ -23,26 +21,17 @@
 
 @contextmanager
 def dataset_manager(name: str) -> Iterator[Dataset]:
-    dataset = get_dataset(name)
-
-    for storage in dataset.get_all_storages():
-        clickhouse = storage.get_cluster().get_query_connection(
-            ClickhouseClientSettings.MIGRATE
-        )
-        for statement in storage.get_schemas().get_drop_statements():
-            clickhouse.execute(statement.statement)
+    from snuba.migrations.migrate import run
+    from snuba.web.views import drop_dataset
 
-        run_storage(storage.get_storage_key())
+    dataset = get_dataset(name)
+    run()
+    drop_dataset(dataset)
```

Comment:
Why is this inverted now?  With respect to what it was before ? Is it because TRUNCATE expects the table to be there ?

```
TRUNCATE TABLE IF EXISTS asdasd

Ok.
```

Reply:
No reason, the order doesn't matter. It seemed very marginally more logical to me to run before dropping.